scripts/kc_data_transfer/src/CsvTransfer.py

```diff
@@ -0,0 +1,87 @@
+import csv
+import logging
+from KeyCloakTransfer import FamWrapper
+import constants
+
+LOGGER = logging.getLogger(__name__)
+
+
+class AppToFam:
```

Comment:
Since this is for csv, could we have documentation to show one example here or the readme.txt (showing maybe a title row - if there is, and the first row data) so that programmers are clear what columns, delimiter and format are expected from the file. Using Python csv, the column order (or index) probably isn't important, so what we can document what column name are expected.

Oh, maybe also specify what Python version this script you are running at would be helpful if in future it does not work for some reason from csv package.

Reply:
Added that in the README, under "Run script -> Note" section, feel free to review

Reply:
Great, thanks for adding that in!